src/asmcnc/apps/systemTools_app/screens/screen_support_menu.py

```diff
@@ -123,8 +124,19 @@
             padding_y: 5.0/800.0*app.width
             border: (0,0,0,0)
 
-        BoxLayout:
-            padding: 0
+        Button:
+            id: button_overwrite_serial_number
+            text: 'Overwrite Serial Number'
```

Comment:
We should be going with something more meaningful for the customer like "Update replacement Z-Head" instead. I'll chat with Az on that.